docs/user/supported_devices.rst

```diff
@@ -354,6 +354,10 @@ ramips-mt7621
 ramips-mt76x8
 -------------
 
+* Cudy
+
+  - WR1000 v1
```

Comment:
Revision in brackets please.

Reply:
Sure, right? This would be the first device in the target with the bracket notation.

> ramips-mt76x8
> -------------
> 
> * Cudy
> 
>   - WR1000 v1
> 
> * GL.iNet
> 
>   - GL-MT300N v2
>   - VIXMINI
> 
> * NETGEAR
> 
>   - R6120
> 
> * TP-Link
> 
>   - Archer C50 v3
>   - Archer C50 v4
>   - TL-MR3020 v3
>   - TL-MR3420 v5
>   - TL-WA801ND v5
>   - TL-WR841N v13
>   - TL-WR902AC v3
> 
> * VoCore
> 
>   - VoCore2

Please consider writing that down as a general rule in the docs.

Reply:
is done since 05432d8dde23da0a84a4d95a57e0c6b306605e4a